dual_net.py

```diff
@@ -210,10 +210,17 @@ def my_res_layer(inputs):
             learning_rate, FLAGS.sgd_momentum).minimize(
                 combined_cost, global_step=global_step)
 
+    policy_label = tf.argmax(labels['pi_tensor'], 1)
+    policy_top1 = tf.argmax(policy_output, 1)
+    policy_top3 = tf.to_int64(tf.nn.top_k(policy_output, 3).indices)
+    top3_match = tf.to_float(tf.reduce_any(tf.equal(tf.expand_dims(policy_label, [1]), policy_top3), 1))
+
     metric_ops = {
-        'accuracy': tf.metrics.accuracy(labels=labels['pi_tensor'],
-                                        predictions=policy_output,
-                                        name='accuracy_op'),
+        'accuracy_top_1': tf.metrics.accuracy(labels=policy_label, predictions=policy_top1),
+        'accuracy_top_3': tf.metrics.mean(top3_match),
```

Comment:
See here for the official way to do top_N statistics

https://github.com/tensorflow/tpu/blob/master/models/official/resnet/resnet_main.py#L346